core/templates/main.html

```diff
@@ -91,7 +80,12 @@
 
         {% endif %}
 
-        scene.renderer.render( scene.scene, scene.camera );
+        function animate() {
+            scene.animate();
+            // requestAnimationFrame(animate);
+        }
+
+        scene.animate();
 		init();
```

Comment:
`scene.animate()` will return the infinite rendering cycle. Please, remove these lines and return 
`scene.renderer.render ( scene.scene, scene.camera );`